server/src/Domain/HumanResource/User/UserAdministrative.entity.ts

```diff
@@ -31,6 +31,9 @@ export class UserAdministrative {
   @Column({ type: 'integer', default: 0, nullable: true })
   private transportFee: number;
 
+  @Column({ type: 'integer', default: 0, nullable: true })
```

Comment:
Puisqu'il y a une valeur par défaut de 0, il y aura toujours une valeur, et on pourrait faire de ce champ un `NOT NULL`, non ? Il faudrait aussi modifier la migration en ce sens.

```suggestion
  @Column({ type: 'integer', default: 0, nullable: false })
```

`transportFee` est dans le même cas, on pourrait imaginer le passer à `NOT NULL` dans une autre PR.

Reply:
Bonne idée, je me note ça pour une future PR